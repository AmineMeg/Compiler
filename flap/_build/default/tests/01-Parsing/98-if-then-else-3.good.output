Line 2, characters 8-10: Error (parsing)
  Syntax error.
