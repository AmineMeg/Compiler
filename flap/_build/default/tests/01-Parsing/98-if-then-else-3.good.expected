Line 3, characters 23-24: Error (parsing)
  Syntax error.
